### PR TITLE
Added CLI download locations

### PR DESCRIPTION
**Pull request by drigodwin**
Added links to the separate CLI download locations

### CODE REVIEW

File: guide/start/running.md [hunk 1]
```diff
@@ -187,10 +187,16 @@ address of the management interface:
...
 </pre>
 
-By default it can be accessed by opening [127.0.0.1:8081](http://127.0.0.1:8081){:target="_blank"} in your web browser. 
+By default it can be accessed by opening [127.0.0.1:8081](http://127.0.0.1:8081){:target="_blank"} in your web browser.
 
 The rest of this getting started guide uses the Apache Brooklyn command line interface (CLI) tool, `br`. 
-This tool is distributed along with Apache Brooklyn. See details on the [Client CLI Reference](../ops/cli/) page. 
+This tool is both distributed with Apache Brooklyn or can be downloaded using the most appropriate link for your OS:
+
+* [Windows](https://www.apache.org/dyn/closer.lua/brooklyn/apache-brooklyn-{{site.brooklyn-version}}-client-cli-windows.zip)
```
Comment (neykov) [inline guide/start/running.md]: Are these actually artifacts uploaded by maven? I thought they were manually created as part of the release process.

Comment (drigodwin) [inline guide/start/running.md]: I believe they will be manually uploaded for the 0.9.0 release.

Comment (neykov) [inline guide/start/running.md]: There are snapshot versions of the docs (for example https://brooklyn.apache.org/v/0.9.0-SNAPSHOT/) so the links won't work there.

**Comment by neykov** [comment]
Merging, thanks @drigodwin

**Pull request merged**